cves/CVE-2013-4444.yml

```diff
@@ -4,13 +4,13 @@ nickname_instructions: |
   Nickname is optional. Provide a useful, professional, and catchy nickname for
   this vulnerability. Ideally fewer than 30 characters. This will be shown
   alongside its CVE to make it more easily distinguished from the rest.
-nickname: Unrestricted File Upload Remote Code Execution
```

Comment:
My upvote on this is 3. I think this is a relevant issue and one easily overlooked as we do not anticipate users to upload malicious files.